src/containers/SidechainHome/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { KeyboardEvent, useRef } from 'react';
+import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { ReactComponent as SidechainLogo } from '../shared/images/sidechain_logo.svg';
+import Header from '../Header';
+import { ANALYTIC_TYPES, analytics } from '../shared/utils';
+import './sidechainhome.css';
+import { ReactComponent as RightArrow } from '../shared/images/side_arrow_green.svg';
+
+interface Props {
+  t: (arg: string) => string;
+}
+
+const SidechainHome = (props: Props) => {
+  const { t } = props;
+
+  const networkText = useRef<HTMLInputElement>(null);
```

Comment:
why the ref?? i believe they are considered a code smell

Reply:
This was the best way I could figure out how to get info to the button on what the input value was.

Reply:
Use `useState` hook for storing `networkText` and update its state on each input `onChange` event. Set `networkText` inside input:
```JS
<input value={networkText}>
```